src/main/webapp/index.js

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import {render} from "react-dom";
+import {App} from "./App";
+
+// Render our App Component to element with ID == target
+const target = 'root';
+render (
+  <App name = 'Sam'/>,
```

Comment:
Did you name this app Sam for testing purposes?

Reply:
Perhaps the name of the prop is ambigious. "name" here is for testing purposes, and refers to a name of a person that is later rendered to screen as the string "Welcome [name]". See _App.js_ also.